### PR TITLE
Added option for enabling DVR in mkcloud run

### PR DESCRIPTION
**Pull request by jsuchome**

### CODE REVIEW

**Comment by jsuchome** [comment]
@aplanas and even my editor (vim :-) ) was showing me that syntax error...

**Comment by aplanas** [comment]
+1 : )

File: scripts/mkcloud [hunk 1]
```diff
@@ -1274,6 +1274,8 @@ Optional
         /root/.ssh/authorized_keys file and to provisioner barclamp
     want_neutronsles12=1 (default 0)
         if there is a SLE12 node, deploy neutron-l3 role into the SLE12 node
+    want_dvr=1 (default='')
```
Comment (vuntz) [inline scripts/mkcloud]: Shouldn't it be `(default 0)`? Like in the other lines above?

Although it seems to be the style you picked in some other lines.

Comment (jsuchome) [inline scripts/mkcloud]: As I'm doing -n test, this looks correct to me... and is also used with other options, yes

**Comment by aspiers** [comment]
+1

File: scripts/qa_crowbarsetup.sh [hunk 2]
```diff
@@ -1700,6 +1700,9 @@ function custom_configuration()
                 if [ "$networkingplugin" = "openvswitch" ] ; then
                     if [[ "$networkingmode" = vxlan ]] || iscloudver 6plus; then
                         proposal_set_value neutron default "['attributes']['neutron']['ml2_type_drivers']" "['gre','vxlan','vlan']"
+                        if [[ -n "$want_dvr" ]]; then
```
Comment (jdsn) [inline scripts/qa_crowbarsetup.sh]: If you use double brackets you can optimize this to:
```
if [[ $want_dvr ]]; then
```

**Comment by jdsn** [comment]
:+1: (even with the -n in double brackets, we will need a big cleanup anyway)

**Pull request merged**